ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OMOJOPipelineModelTestSuite.scala

```diff
@@ -35,44 +35,38 @@ class H2OMOJOPipelineModelTestSuite extends FunSuite with SparkTestContext with
 
   override def createSparkSession(): SparkSession = sparkSession("local[*]")
 
-  test("Test columns names and numbers") {
-    val df =
-      spark.read
-        .option("header", "true")
-        .option("inferSchema", value = true)
-        .csv("examples/smalldata/prostate/prostate.csv")
+  test("Mojo pipeline can be instantiated") {
 
-    val mojoSettings = H2OMOJOSettings(namedMojoOutputColumns = false)
     H2OMOJOPipelineModel.createFromMojo(
+      this.getClass.getClassLoader.getResourceAsStream("mojo2data/pipeline.mojo"),
+      "prostate_pipeline.mojo")
+  }
+
+  test("Mojo pipeline can be saved and loaded") {
+    // Test data
```

Comment:
nit: I think vals could be named just "testData" and "testMojo" instead of adding comments